### PR TITLE
add metrics about rejected logappend msg count

### PR DESCRIPTION
**Pull request by tier-cap**
Signed-off-by: tier-cap <zhengxiaojin@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9498
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 5kbpers — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ This document provides a detailed description of these key metrics on the **TiKV
 - gRPC message error: The number of gRPC message errors per type on each TiKV instance
 - Leader drop: The count of dropped leaders per TiKV instance
 - Leader missing: The count of missing leaders per TiKV instance
+- Log Replication Reject: The number of logappend messages rejected due to insufficient memory on each TiKV instance (update the picture later --to-be-deleted)
```
Comment (Oreoxmt) [inline grafana-tikv-dashboard.md:45]:
```suggestion
- Log Replication Reject: The number of log append messages rejected due to insufficient memory on each TiKV instance
```

**Review by innerr — APPROVED** [review]
LGTM

File: grafana-tikv-dashboard.md (hunk 1 above)
Comment (tier-cap) [reply grafana-tikv-dashboard.md:45]: 'logappend' here means a message type in raft, 'logappend' may be better than 'log append'.

**Review by Oreoxmt — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ This document provides a detailed description of these key metrics on the **TiKV
 - gRPC message error: The number of gRPC message errors per type on each TiKV instance
 - Leader drop: The count of dropped leaders per TiKV instance
 - Leader missing: The count of missing leaders per TiKV instance
+- Log Replication Reject: The number of logappend messages rejected due to insufficient memory on each TiKV instance
 
 ![TiKV Dashboard - Errors metrics](/media/tikv-dashboard-errors.png)
```
Comment (Oreoxmt) [inline grafana-tikv-dashboard.md:47]:
```suggestion
![TiKV Dashboard - Errors metrics](/media/tikv-dashboard-errors-v610.png)
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**